test/object-store/c_api/c_api.cpp

```diff
@@ -288,8 +288,9 @@ class CApiUnitTestTransport : public app::GenericNetworkTransport {
     std::string m_provider_type;
 
 public:
-    CApiUnitTestTransport(const std::string& provider_type = "anon-user")
-        : m_provider_type(provider_type)
+    CApiUnitTestTransport(const std::string& provider_type = {}, uint64_t request_timeout = 60000)
```

Comment:
why the change for the provider_type?  I think the previous default value fit it well

Reply:
I moved the default designation to the member variable initializer (if the string is empty) so I could call this constructor from a different constructor that only takes the `request_timeout` value without having to duplicate the "anon-user" string.

Reply:
This test started failing due to the request_timeout being set to 2500 instead of the default 60000. I don't know why it wasn't failing before...